docs/sdk-and-tools/indices/transactions.md

```diff
@@ -0,0 +1,81 @@
+| fee               | The fee field represents the amount of EGLD the sender paid for the transaction.                                                       |
+| initialPaidFee    | The initialPaidFee field represents the initial amount of EGLD the sender paid for the transaction, before the refund.                 |
+| data              | The data field holds additional information for a transaction. It can either contain a simple message/ESDT related information/etc.    |
+| signature         | The signature field represents the digital information used in cryptocurrency transactions to verify the identity of its participants. |
+| timestamp         | The timestamp field represents the timestamp of the block in which the transaction was executed.                                       |
+| status            | The status field represents the status of the transaction.                                                                             |
+| senderUserName    | The senderUserName field represents the username of the sender address.                                                                |
+| receiverUserName  | The receiverUserName field represents the username of the receiver address.                                                            |
+| hasScResults      | The hasScResults field is true if the transaction has smart contract results.                                                          |
+| isScCall          | The isScCall field is true if the transaction is a smart contract call.                                                                |
+| hasOperations     | The hasOperations field is true if the transaction has smart contract results or logs.                                                 |
+| tokens            | The tokens field contains list of ESDT tokens that are transferred based on the data field.                                            |
+| esdtValues        | The esdtValues field contains list of ESDT values that are transferred based on the data field.                                        |
+| receivers         | The receivers field contains list of receiver addresses in case of ESDTNFTTransfer or MultiESDTTransfer.                               |
+| receiversShardIDs | The receiversShardIDs field contains list of receiver addresses shard IDs.                                                             |
+| type              | The type field represents the type of the transaction based on the data field.                                                         |
+| operation         | The operation field represents the operation of the transaction based on the data field.                                               |
+| isRelayed         | The isRelayed field  is true if the transaction is a relayed transaction.                                                              |
```

Comment:
double white spaces between `field` and `is`

Reply:
removed.